lib/form/paragraph3.flow

```diff
@@ -143,6 +168,9 @@ renderParagraph(
 	words : [ParaWord] = map(list2array(glued), makeParaWord);
 
 	aw = make(0.0);
+	width = if (widthManager == dummyWigiParagraphWidthManager) make(100000.0) else widthManager.width;
```

Comment:
looks like variables width, height and size do nothing. Why do the needed?

Reply:
That is reserve for future PR. Ok, I'll comment it out for a while.